src/solv/cosmo.f90

```diff
@@ -172,7 +175,7 @@ subroutine initCosmo(self, env, num, dielectricConst, nAng, radScale, vdwRad, &
    !> Offset for surface integration cutoff
    real(wp), intent(in) :: rOffset
 
-   integer :: igrid, stat, iat, izp
+   integer :: igrid, stat, iat, izp, irad
```

Comment:
Is irad really used?

Reply:
Seems like a coding artifact. This can be deleted.